absl/time/time.h

```diff
@@ -76,9 +76,9 @@ struct timeval;
 #endif
 #include <chrono>  // NOLINT(build/c++11)
 
-#ifdef __cpp_impl_three_way_comparison
+#ifdef __cpp_lib_three_way_comparison
```

Comment:
This requires `#include <version>` for some compilers (which is annoying in my opinion).